tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py27,py34
 skipsdist = True
 
 [testenv]
-recreate = True
+recreate = False
```

Comment:
I think there was a reason @chosak left it as `True` in the main testenv when he made the "fast" env below, wasn't there?